src/jacobian.jl

```diff
@@ -105,16 +105,25 @@ function build_jac_and_jac_config(alg, f::F1, uf::F2, du1, u, tmp, du2) where {F
 end
 
 # Build Jacobian Caches
-function jacobian_caches(alg::Union{NewtonRaphson, LevenbergMarquardt, TrustRegion}, f, u,
+function jacobian_caches(alg::Union{NewtonRaphson, LevenbergMarquardt, TrustRegion,PseudoTransient}, f, u,
     p, ::Val{true})
     uf = JacobianWrapper(f, p)
 
     du1 = zero(u)
     du2 = zero(u)
     tmp = zero(u)
     J, jac_config = build_jac_and_jac_config(alg, f, uf, du1, u, tmp, du2)
+    
+    if alg isa PseudoTransient
+        alpha = convert(eltype(u),alg.alpha_initial)
+        J_new = J - (1/alpha) * I
+        linprob = LinearProblem(J_new, _vec(zero(u)); u0 = _vec(zero(u)))
```

Comment:
What difference does this make?

Reply:
If `J` is a VecJac operator, then the addition constructs an `AddedOperator`. needed for Krylov Subspace compatibility.

Reply:
Without this the types wont match in `dolinsolve`